infra/cifuzz/cifuzz.py

```diff
@@ -151,7 +151,7 @@ def build_fuzzers(project_name,
   if helper.docker_run(command):
     logging.error('Building fuzzers failed.')
     return False
-  return True
+  return not helper.check_build_impl(project_name, None)
```

Comment:
Can we call this from the caller of build_fuzzers?

Reply:
Want to keep the code modular.

Reply:
Done.